### PR TITLE
Fix Windows build (in Debug config for now).

### PR DESCRIPTION
**Pull request by speps**
All tests except one now pass in Debug config. The only failing test is this one:

```
-------------------------------------------------------------------------------
XTree<3> cancellation
-------------------------------------------------------------------------------
libfive\libfive\test\xtree.cpp(212)
...............................................................................

libfive\libfive\test\xtree.cpp(237): FAILED:
  REQUIRE( elapsed_ms.count() < 150 )
with expansion:
  217 < 150

===============================================================================
test cases:   91 |   90 passed | 1 failed
assertions: 8616 | 8615 passed | 1 failed
```

### CODE REVIEW

File: libfive/src/render/brep/xtree.cpp [hunk 1]
```diff
@@ -158,7 +158,7 @@ XTree<N>::XTree(XTreeEvaluator* eval, Region<N> region,
                 for (unsigned i=0; i < children.size(); ++i)
                 {
                     futures[i] = std::async(std::launch::async,
-                        [&eval, &rs, i, min_feature, max_err, &cancel]()
+                        [&eval, &rs, &cancel, i, min_feature, max_err]()
```
Comment (speps) [inline libfive/src/render/brep/xtree.cpp]: Yes this is actually fixing a crash...

File: CMakeLists.txt [hunk 2]
```diff
@@ -21,10 +21,10 @@ else()
     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_BINARY_DIR}/Output/Debug)
     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_BINARY_DIR}/Output/Release)
     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
-    set(CMAKE_CXX_FLAGS "/EHsc /WX /D_USE_MATH_DEFINES")
+    set(CMAKE_CXX_FLAGS "/EHsc /WX /D_USE_MATH_DEFINES /D_SCL_SECURE_NO_WARNINGS")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267 /wd4244 /wd4305")
-    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
-    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MD")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MDd")
```
Comment (speps) [inline CMakeLists.txt]: This means the VC runtime is now required but also fixes a crash due to memory allocation in DLLs.

File: libfive/src/render/brep/xtree.cpp (hunk 1 above)
Comment (mkeeter) [reply libfive/src/render/brep/xtree.cpp]: Well, that's terrifying 😆

**Comment by mkeeter** [comment]
Thanks!

**Pull request merged**